README.md

```diff
@@ -42,7 +42,6 @@ On Google Chrome:
 7. Fill in the form with your API key and workspace name (the extension uses the prod API by default), and save;
 8. Go to a GitHub repository corresponding to your workspace settings, open an issue or a pull request and voilà!
```

Comment:
The steps `7.` and `8.` can be updated to something like:

7. Sign in with your Orbit account and select a workspace;
8. Go to any GitHub issue/PR/Discussion, LinkedIn profile, Twitter profile, or Gmail email;
9. You should see the Orbit widget: you’re all set!

Reply:
😱 Exellent spot, thanks! Missed this